<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKAutoResponseItem</class>
 <widget class="QWidget" name="SAKAutoResponseItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加自动回复设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="labelInfo">
     <property name="text">
      <string>收发数据使用16进制格式文本表示，可使用“菜单-&gt;工具-&gt;16进制格式文本生成”工具生成。</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditReceiveData">
     <property name="text">
      <string>00 11 22 33 44 55 66 77 88 99 0A 0B 0C 0D 0E 0F</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditSendData">
     <property name="text">
      <string>0F 0E 0D 0C 0B 0A 09 08 07 06 05 04 03 02 01 00</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>接收内容</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>回复内容</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>附加说明</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditDescription">
     <property name="text">
      <string>心跳包</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonApply">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonCancle">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
